CIP-onchain-tx-chaining/README.md

```diff
@@ -0,0 +1,198 @@
+---
+CIP: _???
+Title: onchain tx chaining
+Category: Ledger | Plutus
+Status: Draft
+Authors:
+    - Michele Nuzzi <michele.nuzzi.2014@gmail.com>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/?
+Created: 2023-04-13
+License: CC-BY-4.0
+---
+
+<!-- Existing categories:
+
+- Meta                   | For meta-CIPs which typically serves another category or group of categories.
+- Reward-Sharing Schemes | For CIPs discussing the reward & incentive mechanisms of the protocol.
+- Wallets                | For standardisation across wallets (hardware, full-node or light).
+- Tokens                 | About tokens (fungible or non-fungible) and minting policies in general.
+- Metadata               | For proposals around metadata (on-chain or off-chain).
+- Tools                  | A broad category for ecosystem tools not falling into any other category.
+- Plutus                 | Changes or additions to Plutus
+- Ledger                 | For proposals regarding the Cardano ledger
+- Catalyst               | For proposals affecting Project Catalyst / the Jörmungandr project
+
+-->
+
+## Abstract
+<!-- A short (\~200 word) description of the proposed solution and the technical issue being addressed. -->
+
+Thanks to the transaction determinism, in Cardano is possible to chain transactions so that new transaction can be created using transaction outputs of transactions that have not been included in the blockchain yet.
+
+Despite being extremly powerful this process as currently some limitations:
+- it gives no guarantees that all of the multiple transactions are valid
+- the process happens entrierly offchain; without the possibility for a plutus script to be aware of it
```

Comment:
This last paragraph shall actually be part of the "Motivation" section IMO as it gives some clear justification for _why is this CIP necessary_.

Reply:
Will do!